kafka-bin-scripts/README.adoc

```diff
@@ -217,13 +189,14 @@ ifndef::qs[]
 endif::[]
 
 [id="proc-consuming-messages-kafka-bin-scripts_{context}"]
-== Consuming messages using Kafka bin scripts
+== Consuming messages using Kafka scripts
 
 You can use the `kafka-console-consumer` script to consume messages from Kafka topics. This example consumes the messages that you sent previously with the producer that you created with the `kafka-console-producer` script.
 
 .Prerequisites
 
 * You used the `kafka-console-producer` script to produce example messages to a topic.
+* You have checked that the `kafka-console-consumer` script is in the `bin/` directory of your Kafka download.
```

Comment:
Not sure we need this. There was something similar in the download step.

Reply:
No need

Reply:
Removed